packages/fxa-auth-server/README.md

```diff
@@ -1,148 +1,26 @@
 
-[Overview](docs/overview.md)
+- user account details, such as email address and password
+- the list of devices connected to the account
+- the master encryption keys used by services connected to the account
 
-[Detailed design document](https://github.com/mozilla/fxa-auth-server/wiki/onepw-protocol)
+There is no UI provided by this server. It is expected that user interaction will happen through Firefox or a hosted website, which will use the API provided by this server.
 
-[Detailed API spec](https://mozilla.github.io/ecosystem-platform/api)
+## Concepts
 
-[Guidelines for Contributing](CONTRIBUTING.md)
+Each user of the service creates an **account** which has a unique id. Access to the account is authenticated by an email/password pair and uses the [Secure Remote Password protocol](https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol) to avoid revealing the password to the server. Note that our SRP protocol ordering is slightly different from the sample on wikipedia, but the variables are the same.
 
-## Prerequisites
+The user connects one or more **devices** to their account. Each device performs the SRP authentication protocol to obtain one or more opaque **authentication tokens**, which are then used to make [Hawk signed requests](https://github.com/mozilla/hawk/) when interacting with the server API. This token exchange means the device does not need to keep the the user's password in persistent storage.
 
-- node 6+
-- npm 2
-- Grunt
-- postfix
-- memcached
-- redis
+Once connected, each device can fetch the user's **encryption keys**. The server maintains two keys for each account, called **kA** and **kB**. kA is known to the server, and is intended for encrypting data that must be recoverable in the event of password reset. kB is stored encrypted by the user's password, and is intended for more secure encryption at the cost of unrecoverability when the password is forgotten.
 
-## Install
+## API
 
-On some systems running the server as root will cause working directory permissions issues with node. It is recommended that you create a separate, standard user to ensure a clean and more secure installation.
-
-Clone the git repository and install dependencies:
-
-    git clone https://github.com/mozilla/fxa-auth-server.git
-    cd fxa-auth-server
-    npm install
-
-This runs a script `scripts/start-local.sh` as defined in `package.json`. This will start up
-4 services, three of which listen on the following ports (by default):
-
-- `bin/key_server.js` on port 9000
-- `test/mail_helper.js` on port 9001
-- `./node_modules/fxa-customs-server/bin/customs_server.js` on port 7000
-
-When you `Ctrl-c` your server, all 4 processes will be stopped.
-
-To start the server in dev MySQL store mode (ie. `NODE_ENV=dev`), run:
-
-    npm run start-mysql
-
-To forward Stripe webhooks to your local, you can use the [Stripe CLI](https://stripe.com/docs/stripe-cli).
```

Comment:
Moved to https://mozilla.github.io/ecosystem-platform/tutorials/development-setup#stripe